src/components/navbar/navbar.component.tsx

```diff
@@ -87,7 +87,10 @@ const Navbar: React.FC<NavbarProps> = ({
                 <Logo />
               </HeaderLink>
               <HeaderGlobalBar>
-                <ExtensionSlot extensionSlotName="top-nav-actions-slot" />
+                <ExtensionSlot
+                  style={{ display: "flex" }}
```

Comment:
you can probably add a new css class and append it like a css module.
```
className={styles.topNavContainer} 
```

or something like that

Reply:
hi seankwon, I have tried it but it is giving this
![className Attribute error 497](https://user-images.githubusercontent.com/58779460/113433158-4b578280-93fc-11eb-8f53-eeedd332eb93.png)
 error

Reply:
Hey @nk183, please checkout my PR, #67 , you can do it the way I did it. I didn't know you are working on it, so I pushed my changes. You can do the way I did it and I'll close mine then!

Reply:
Wrap the ExtensionSlot with div and assign the class to that div.

Reply:
No and please stop giving wrong advice @vasharma05 (I know you mean no harm, but it would be better to first look for the real solution before giving some workaround / semi-wise recommendations).

https://github.com/openmrs/openmrs-esm-core/blob/master/packages/esm-react-utils/src/ExtensionSlot.tsx#L29

`className` definitely works and is also actively used on some places (e.g., https://github.com/openmrs/openmrs-esm-patient-chart/blob/feature/monorepo/packages/esm-patient-chart-app/src/view-components/grid-view.component.tsx#L57).

Reply:
Oh, my bad. I'm still learning about ExtensionSlots. Thank You @FlorianRappl !

Reply:
hi @FlorianRappl , I checked your links that but in esm-primary-navigation it is giving the above error can you please guide me where should check to address the root cause of this error

Reply:
Not sure what problem you are facing @nk183. Make sure that you use the latest version of the core framework.

Reply:
oh! thanks @FlorianRappl I was not using the latest version of esm framework now it is working fine

Reply:
Hi @nk183, What's up with this ? Have you succeeded ? The problem is still visible. I wanted to create an Issue already concerning this. Please try to push your changes before some else does.

Reply:
I have done just a pending check of change because
 of navbar not visible

Reply:
Navbar not visible ? I don't understand please. Which module are you running ? to see the navbar, run the esm-primary-navigation module.